src/midi.js

```diff
@@ -3,7 +3,14 @@
 const fs = require('fs');
 const assert = require('assert');
 const jsmidgen = require('jsmidgen');
-
+var transposition = 0;
+/**
+ * Takes an integer and transposes all notes to a different middle C octave.
+ * @param {Integer} octaveIndex		The new octave for middle C.
+ */
+function setMiddle(octaveIndex){
+	transposition = octaveIndex-4;
```

Comment:
Add a space on the either side of the hypen. `octaveIndex - 4;` instead of `octaveIndex-4;`